runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -50,6 +55,25 @@
 {
 	private static final int NUM_MAPDOTS = 6;
 
+	private static List<WidgetInfo> orbs = ImmutableList
```

Comment:
```
ImmutableList.of(
    WidgetInfo.MINIMAP_HEALTH_ORB,
    WidgetInfo.MINIMAP_PRAYER_ORB,
    WidgetInfo.MINIMAP_RUN_ORB,
    WidgetInfo.MINIMAP_SPEC_ORB
);
```

Reply:
Also, this should be `private static final`, and var name should be uppercased to match.